device-types/ADVA/FSP-150-CM.yaml

```diff
@@ -0,0 +1,63 @@
+---
+manufacturer: ADVA
+model: FSP-150-CM
+slug: adva-fsp-150-cm
+part_number: '1044004001'
+u_height: 4
+is_full_depth: false
```

Comment:
Please add the `airflow` data point if you have it or can find it. Often these are listed in the datasheet, but sometimes they are not